api/pkg/apis/v1alpha1/vendors/federation-vendor.go

```diff
@@ -318,6 +318,11 @@ func (f *FederationVendor) onSync(request v1alpha2.COARequest) v1alpha2.COARespo
 		if count == "" {
 			count = "1"
 		}
+		scope, exist := request.Parameters["scope"]
```

Comment:
This call is to synchronize catalogs from parent site to child site. Currently we maintain catalogs queue per site rather than per namespace per site. So I think scope should not be passed in from request parameters. Scope should be stored inside Catalog and once Catalogs are sync to child site, then child site can still tell which namespace the catalog belongs to.

Reply:
I believe according to the email thread we expect a child to have the same namespaces as the parent. But I can revert it back to default as it used to be.